staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go

```diff
@@ -197,9 +199,12 @@ func (w *WebhookAuthorizer) Authorize(attr authorizer.Attributes) (decision auth
 			w.responseCache.Add(string(key), r.Status, w.unauthorizedTTL)
 		}
 	}
-	if r.Status.Allowed {
+	switch {
+	case r.Status.Denied:
```

Comment:
The comment on these fields states

> Denied may not be true if Allowed is true.

Failing closed is good, but I think we should treat this as an error case.

Reply:
Done.